src/index.html

```diff
@@ -7,8 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="top">
+      <div class="logos">
+        <a href="#home">
+          <img
+            class="logo"
+            src="images/logo.png"
+            alt="logoimage"
+          >
+        </a>
+      </div>
+
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active underline"
+              href="#Apple">
```

Comment:
fix your tag formatting, fix it everywhere
```suggestion
            <a class="nav__link is-active underline" href="#Apple">
```